docs/howto/openstack/neutron/create-security-groups.md

```diff
@@ -0,0 +1,272 @@
+    | c105b4640 |             |           |           |            |           |                       |                      |
+    | 72b       |             |           |           |            |           |                       |                      |
+    | 86b9413a- | None        | IPv6      | ::/0      |            | ingress   | None                  | None                 |
+    | ad23-     |             |           |           |            |           |                       |                      |
+    | 46c4-     |             |           |           |            |           |                       |                      |
+    | a35e-     |             |           |           |            |           |                       |                      |
+    | 9306945dc |             |           |           |            |           |                       |                      |
+    | 63c       |             |           |           |            |           |                       |                      |
+    | ad4a19ef- | None        | IPv6      | ::/0      |            | egress    | None                  | None                 |
+    | 7fab-     |             |           |           |            |           |                       |                      |
+    | 4eba-     |             |           |           |            |           |                       |                      |
+    | 9982-     |             |           |           |            |           |                       |                      |
+    | e5b109be1 |             |           |           |            |           |                       |                      |
+    | 21c       |             |           |           |            |           |                       |                      |
+    | f53b1a12- | None        | IPv4      | 0.0.0.0/0 |            | egress    | None                  | None                 |
+    | edbb-     |             |           |           |            |           |                       |                      |
+    | 480b-     |             |           |           |            |           |                       |                      |
+    | 910b-     |             |           |           |            |           |                       |                      |
+    | a71c48363 |             |           |           |            |           |                       |                      |
+    | 46f       |             |           |           |            |           |                       |                      |
+    +-----------+-------------+-----------+-----------+------------+-----------+-----------------------+----------------------+
+    ```
+
+    The IDs of the two ingress rules, one for ipv4 traffic and one for ipv6, in this case are: `5e5e9f4d-1faa-492d-91f1-c105b464072b` and `86b9413a-ad23-46c4-a35e-9306945dc63c`
```

Comment:
Capitalization: "IPv4" and "IPv6", please.

Reply:
Thank you Florian! fixed.